src/pages/contact.js

```diff
@@ -0,0 +1,43 @@
+import Link from 'next/link'
+import { useUser } from '@auth0/nextjs-auth0/client'
+
+export default function ContactPage() {
+    const { user } = useUser();
+
+    return (
+        <>
+            <main className="bg-[url('../rodrigo-s-2mz9IKab7DE-unsplash.jpg')] bg-cover bg-center h-screen" data-testid="">
+                <div className="w-full bg-sky-300 bg-opacity-90">
+                    <h1 className="grid h-40 place-items-center text-white text-7xl">Contact Us</h1>
+                </div>
+                <div className="flex bg-black bg-opacity-75 h-10">
+                    <Link href='/' className="hover:text-sky-200 text-white mt-2 ml-4">Home</Link>
+                    <div className='flex ml-auto mr-4'>
+                        {!user && <Link href="/api/auth/login" className="hover:text-sky-200 text-white ml-4 mt-2">Login</Link>}
+                        {user && <Link href="/api/auth/logout" className="hover:text-sky-200 text-white mt-2 ml-4">Logout</Link>}
+                    </div>
+                </div>
```

Comment:
the above code is repeated code in both files, move it to app.js

You can get access to the page by looking at the url to find out is you need to show contact us